go.mod

```diff
@@ -3,34 +3,35 @@ module sigs.k8s.io/gcp-filestore-csi-driver
 go 1.18
 
 require (
-	cloud.google.com/go/compute/metadata v0.2.1
-	github.com/container-storage-interface/spec v1.5.0
+	cloud.google.com/go/compute/metadata v0.2.3
```

Comment:
rebase on top of https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/pull/448